README.md

```diff
@@ -2,44 +2,21 @@
 
 The Ziggurat implementation for XRPLF's `rippled` nodes.
 
+## Prerequisites
+Ziggurat is written in stable Rust; you can install the Rust toolchain by following the official instructions [here](https://www.rust-lang.org/learn/get-started)
+
 ## Getting started
 
+### Preconditions
 1. Clone this repository.
 2. Build [rippled](https://github.com/XRPLF/rippled) from source.
-3. Create the `~/.ziggurat/ripple/setup` directories, and copy the `setup/validators.txt` file there.
-   ```
-   cp setup/validators.txt ~/.ziggurat/ripple/setup
-   ```
-4. In the same directory create a `config.toml` with the following contents:
-   ```
-   path = "<path to the directory where you built rippled>"
-   start_command = "./rippled"
-   ```
-5. Create a package of IP addresses which are required for performance tests. From the root repository directory run, e.g.:
-   Under Linux (to generate dummy devices with addresses):
-   ```
-   sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev_prefix test_zeth
-   ```
-   Under MacOS or Linux (to add whole subnet to loopback device - under Linux: lo, MacOS: lo0):
-   ```
-   sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev lo0
-   ```
-   Read ./tools/ips.py for more details.
-6. Run tests with:
-   ```
-   cargo +stable t -- --test-threads=1
-   ```
-
-### Initial state
-Specific tests require an initial node state to be set up.
-Follow the steps below to save an initial state that can be loaded later in certain tests.
 
-#### Preparation (needs to be done once)
-1. Make sure you have python3 installed. You should be able to run `python3 --version`.
-2. Install `xrpl` python lib: `pip3 install xrpl-py`.
+#### Running setup script
+3. Make sure you have python3 installed. You should be able to run `python3 --version`.
+4. Install `xrpl` python lib: `pip3 install xrpl-py`.
```

Comment:
`brew install coreutils `

for Mac only

Reply:
Done :white_check_mark: